### PR TITLE
Exporting

### PR DESCRIPTION
**Pull request by pkukulak**
Added "Export to PDF" to the navbar, which directs to a new response page. It currently displays a picture of my cat :)

### CODE REVIEW

**Comment by david-yz-liu** [comment]
@pkukulak Nice work so far! Let me know if you're having trouble generating the PDF.

**Comment by pkukulak** [comment]
@david-yz-liu Generating the PDF is no problem; the library comes with a pretty thorough test file that illustrates what needs to be done. I'm hoping to have a download link on the page shortly.

**Comment by david-yz-liu** [comment]
@pkukulak Great!

**Comment by Ian-Stewart-Binks** [comment]
Nice @pkukulak !

File: hs/ExportResponse.hs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+exportResponse =
+   ok $ toResponse $
+    masterTemplate "Courseography - SVG serving test!"
+                [H.meta ! A.name "keywords"
+                        ! A.content "",
+                 aboutLinks
+                ]
+                (do
+                    header "export"
+		    exportHtml
```
Comment (Ian-Stewart-Binks) [inline hs/ExportResponse.hs]: We haven't exactly set any constraints on `do` block style, and quite frankly I think I've made my own up somehow (which is probably a bad thing), but please align these two lines.

**Comment by pkukulak** [comment]
Yeesh. Didn't even notice. Fixing now.

**Comment by pkukulak** [comment]
This is turning out to be quite difficult. I think that Haskell is my primary obstacle at the moment. I will need more time to work this out.

**Pull request merged**